main.lua

```diff
@@ -88,6 +89,26 @@ local function sleep(ms)
     end
 end
 
+local function setCursorPos(x, y)
+    io.write(string.char(27) .. "[" .. y .. ";" .. x .. "H")
```

Comment:
trzeba bedzie zrobic globalna zmienna, typu escape_char bo to wszedzie jest uzywane

Reply:
No też to zauważyłem i zgadzam się z tym